client/my-sites/customer-home/components/celebrate-launch-modal.jsx

```diff
@@ -77,7 +98,7 @@ function CelebrateLaunchModal( { setModalIsOpen, site, allDomains } ) {
 			);
 			buttonText = translate( 'Claim your free domain' );
 			buttonHref = `/domains/add/${ site.slug }`;
-		} else if ( hasCustomDomain ) {
+		} else if ( hasCustomDomain || isA4ASite ) {
```

Comment:
Instead of removing the mentions of a custom domain, we can probably updating https://github.com/Automattic/wp-calypso/pull/90866/files#diff-b2a3bdf3b4cec65b339e0803b39186ea50d15861cfa347b97294688a3b89e58eR69 to ensure we render the upsell banner for A4A sites!

Reply:
Good point 👍